vibesec/global.setup.ts

```
@@ -0,0 +1,40 @@
+import { clerk, clerkSetup } from '@clerk/testing/playwright';
+import { test as setup } from '@playwright/test';
+import path from 'path';
+
+// Configure Playwright with Clerk
+setup('global setup', async ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)